pages/golfers/[userId].js

```diff
@@ -1,24 +1,16 @@
-import { useState, useEffect } from 'react'
+import { useRouter } from 'next/router'
 import useUsersScore from '../../lib/useUsersScore'
 import Layout from '../../components/Layout'
-import { getUserId, getUsername } from '../../lib/userAuth'
 
 function GolfersDetails() {
-
-  const [ userName, setUsername ] = useState('')
-  const [ userId, setUserId ] = useState('')
+  const router = useRouter()
+  const { userId } = router.query
   const { scores } = useUsersScore(userId)
 
-  useEffect(() => {
-    setUserId(getUserId())
-    setUsername(getUsername())
-  }, [])
-
   return (
     <Layout>
       <div className="flex items-center flex-col">
-        <h1 className="text-2xl my-8">{`Hello ${userName}`}</h1>
-        <p className="p-5 text-xl">Here are your scores:</p>
+        <p className="p-5 text-xl">Here are {scores && scores[0].user_name} scores:</p>
```

Comment:
What if the user doesn't have any scores, are we not going to display anything here? 😄

Reply:
<img width="1011" alt="Screenshot 2023-08-28 at 18 14 50" src="https://github.com/IuliaVele/golfr_frontend/assets/141643253/6c75c803-4d7d-4086-8f1c-862f58509853">
This will be the display for an user that has no scores. 😄